src/app/demo/feedback-page/feedback-page.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component } from '@angular/core';
+import { AngularFireDatabase } from 'angularfire2/database';
+import {Message} from "../../feedback/message";
+
+@Component({
+  selector: 'app-feedback-page',
+  templateUrl: './feedback-page.component.html',
+  styleUrls: ['./feedback-page.component.css']
+})
+export class FeedbackPageComponent {
+  constructor(private database: AngularFireDatabase) {
+  }
+  rawFeedbackList: Array<any> = [];
+  currentFeedbackList: Message[] = [];
+  dates: Array<string> = [''];
+  selectedDate = '';
+  feedbackersCount = {};
+  feedbackersNames = [];
+
+
+  ngOnInit() {
+    this.database.list('/feedback').subscribe(list => {
+      this.feedbackersCount = {};
+      this.feedbackersNames = [];
+      list.forEach((val) => {
+        this.feedbackersCount[val.name] = this.feedbackersCount[val.name] ? this.feedbackersCount[val.name] + 1 : 1;
+      });
+      this.feedbackersNames = Object.keys(this.feedbackersCount);
+      this.feedbackersNames.sort((a,b) => {
+        return this.feedbackersCount[b] - this.feedbackersCount[a];
+      });
+      this.rawFeedbackList = list;
+      this.currentFeedbackList = this.rawFeedbackList;
+    });
+  }
+
+  isDone(eleKey: any, event): void{
+    this.database.object('/feedback/'+eleKey).update({
+      isDone: event.target.checked
+    });
+  }
+
+  isShown = (isDone) => {
+    return true;
```

Comment:
What is this function do? It has an argument but always returns true? Can this just be a const?